content/en/user-guide/cloud-pods/pods-cli/index.md

```diff
@@ -54,100 +110,308 @@ $ localstack pod save --name my-pod --visibility public
+The `versions` command lists all the available versions of a Cloud Pod. 
+The CLI manual for the `version` command is as follows:
+
+### `remote`
 
-The `list` command lists all of the available Cloud Pods. It shows all the pods available for a single user and its organization by default. The CLI manual for the `list` command is as follows:
+The remote command lets you manage remotes different from the default one.
+It offers 3 commands, i,e.,: `add`, `delete`, and `list`.
+
+For more info about remote usage, check our [documentation](https://docs.localstack.cloud/user-guide/cloud-pods/remotes/).
 
 {{< command >}}
-$ localstack pod list --help
 <disable-copy>
-Usage: localstack pod list [OPTIONS] [REMOTE]
+Usage: pod remote [OPTIONS] COMMAND [ARGS]...
+
+  Manage cloud pod remotes
 
 Options:
-  -p, --public               List all the available public Cloud Pods
-  -f, --format [table|json]  The formatting style for the list pods command output.  [default:
-                             table]
-  -h, --help                 Show this message and exit.
+  --help  Show this message and exit.
+
+Commands:
+  add     Add a remote
+  delete  Delete a remote
+  list    Lists the available remotes
 </disable-copy>
 {{< / command >}}
 
-### versions
+#### `remote add`
+
+{{< command >}}
+<disable-copy>
+Usage: pod remote add [OPTIONS] NAME URL
+
+  Add a new remote for Cloud Pods.
 
-The `versions` command lists all the available versions of a Cloud Pod. The CLI manual for the `version` command is as follows:
+  A remote is the place where your Cloud Pods are stored. By default, Cloud
+  Pods are store in the LocalStack platform.
 
+Options:
+  --help  Show this message and exit.
+</disable-copy>
+{{< / command >}}
+
+#### `remote delete`
+
+{{< command >}}
+<disable-copy>
+Usage: pod remote delete [OPTIONS] NAME
+
+  Remove a remote for Cloud Pods.
+
+Options:
+  --help  Show this message and exit.
+</disable-copy>
+{{< / command >}}
+
+#### `remote list`
+{{< command >}}
+<disable-copy>
+Usage: pod remote list [OPTIONS]
+
+Options:
+  -f, --format [table|json]  The formatting style for the remotes command
+                             output.
+
+  --help                     Show this message and exit.
+</disable-copy>
+{{< / command >}}
+
+---
+
+# Local Commands
+
+In alternative to the commands in the `pod` group, we also offer a simple alternative to save and load the LocalStack state.
+The `state` group offers two commands to export and import the state from the LocalStack container to/from a zip file from the host machine.
+
+## `state` syntax
 {{< command >}}
-$ localstack pod versions --help
 <disable-copy>
-Usage: localstack pod versions [OPTIONS] NAME
+Usage: state [OPTIONS] COMMAND [ARGS]...
+
+  (Beta) Manage and manipulate the localstack state.
+
+  The state command group allows you to interact with LocalStack's state
+  backend.
+
+  Read more: https://docs.localstack.cloud/references/persistence-
+  mechanism/#snapshot-based-persistence
```

Comment:
I understand that the CLI auto-wraps these long lines, but we could consider printing them out in full here..
```suggestion
  Read more: https://docs.localstack.cloud/references/persistence-mechanism/#snapshot-based-persistence
```